charts/osm/templates/osm-deployment.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   labels:
     app: osm-controller
     meshName: {{ .Values.OpenServiceMesh.meshName }}
+    {{ if .Values.OpenServiceMesh.enforceSingleMesh }}enforceSingleMesh: "true"{{ end }}
```

Comment:
This should always be set to true, we don't need the condition

Reply:
in the osm repo enforceSingleMesh is false in the values.yaml. Should we set it to true here?

Reply:
@ritazh